src/components/SignIn.jsx

```diff
@@ -49,17 +57,24 @@ class SignIn extends React.Component {
   render() {
     const { email, password } = this.state;
     return (
-      <form>
+      <form style={sectionStyle}>
```

Comment:
If we bring the styling out to the js file, this because a class rather than a style.